workflow/sample_files/base-bldgtype-multifamily-adjacent-to-multifamily-buffer-space.xml

```diff
@@ -107,7 +107,7 @@
             <WallType>
               <WoodStud/>
             </WallType>
-            <Area>685.9</Area>
+            <Area>783.8</Area>

```

Comment:
If we proceed with this fix, I suggest you change the aspect ratio provided to generate these sample files so that they don't change. (The sample files were created prior to the Build measure existing and being used to generate them.)

Reply:
You mean change the aspect ratio to 1 / 1.8 = 0.6, or whatever value produces unchanged sample files?

Reply:
Yep.